src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -0,0 +1,101 @@
+@import '~amo/css/styles';
+
+.GuidesAddonCard {
+  display: flex;
+  flex-direction: column;
+
+  // stylelint-disable max-line-length
+  // TODO: check on get-firefox button style.
+  // See https://github.com/mozilla/addons-frontend/issues/6432#issuecomment-433083079.
+  .Button--get-firefox,
+  .AMInstallButton {
+    @include respond-to(medium) {
+      align-self: center;
+      margin-left: auto;
+    }
+  }
+
+  .Icon-trophy {
+    height: 12px;
```

Comment:
The `height` and `width` feel like they should go in `Icon/styles.scss` in the `.Icon-trophy` class. The margin can stay here, though.